lib/smart_answer_flows/coronavirus-business-reopening/outcomes/_social_distancing.erb

```diff
@@ -24,6 +24,20 @@ Where you cannot stay 2 metres apart (or 1 metre with risk mitigation where 2 me
 - work side by side or back-to-back rather than face-to-face
 - have fixed teams to minimise exposure
 
+<% if calculator.sector?("hospitality") %>
+### Managing service of food and drink
```

Comment:
This is being conditionally shown is somewhat inconsistent with the other sector specific sections e.g.
- Restaurants with takeaway and delivery: social distancing
- Labs and research facilities: social distancing
- etc..